src/views/RecIM/components/List/Listing/Listing.module.scss

```diff
@@ -1,3 +1,4 @@
+@use 'sass:color';
```

Comment:
What is this used for?

Reply:
Not used anymore, it was originally used to pass a weird linting check where we couldn't use `mix`. I wanted to make a specific hover color which is why you see the residual changes on blue-tint and shades. I'll remove it.